doc/source/examples/incompressible-flow/2d-transient-flow-around-cylinder/2d-transient-flow-around-cylinder.rst

```diff
@@ -0,0 +1,266 @@
+
+.. code-block:: text
+
+   subsection mesh adaptation
+       set type                   = kelly
+       set variable               = pressure
+       set fraction type          = number
+       set max number elements    = 70000
+       set max refinement level   = 3
+       set min refinement level   = 1
+       set frequency              = 1
+       set fraction refinement    = 0.02
+       set fraction coarsening    = 0.01
+   end
+
+Here, we are using the pressure as the variable for the `kelly error estimator <https://lethe-cfd.github.io/lethe/parameters/cfd/mesh_adaptation_control.html>`_, unlike the previous examples which were using the velocity. Additionally, the ``fraction refinement`` and ``fraction coarsening`` are set to lower values than the previous examples (i.e., respectively 0.02 and 0.01) to enable a gradual growth of the mesh size.
+
+
+Initial and boundary conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The `Initial Condition <https://lethe-cfd.github.io/lethe/parameters/cfd/initial_conditions.html>`_ and `Boundary Conditions <https://lethe-cfd.github.io/lethe/parameters/cfd/boundary_conditions_cfd.html>`_ are defined as in `2D Flow around a cylinder <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.html>`_.
+
+.. code-block:: text
+
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+                set Function expression = 1; 0; 0
+        end
+    end
+
+    subsection boundary conditions
+        set number                  = 3
+        subsection bc 0
+            set type              = noslip
+        end
+        subsection bc 1
+            set type              = function
+            subsection u
+                set Function expression = 1
+            end
+            subsection v
+                set Function expression = 0
+            end
+            subsection w
+                set Function expression = 0
+            end
+        end
+        subsection bc 2
+            set type              = slip
+        end
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+The Reynolds number must be high enough to capture a transient flow and study the evolution of the drag and lift coefficients in time. Therefore, we set Re = 200 through the value of the kinematic viscosity in the same manner as for the `2D Lid-driven cavity flow <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-lid%E2%80%90driven-cavity-flow/lid%E2%80%90driven-cavity-flow.html>`_. Since :math:`U_\infty = 1` and the :math:`D = 1`, we have :math:`Re=\frac{1}{\nu}`, where :math:`\nu` is the kinematic viscosity.
```

Comment:
```suggestion
The Reynolds number must be high enough to create a transient flow and evolution of the drag and lift coefficients in time. Therefore, we set Re = 200 through choosing a kinematic viscosity in the same manner as for the `2D Lid-driven cavity flow <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-lid%E2%80%90driven-cavity-flow/lid%E2%80%90driven-cavity-flow.html>`_. Since :math:`U_\infty = 1` and the :math:`D = 1`, we have :math:`Re=\frac{1}{\nu}`, where :math:`\nu` is the kinematic viscosity.
```